Merge pull request #9 from BoG-Dev-Bootcamp-F23/ishaan

Ishaan-Ali opened the pull request.

Commit:
Created ankith branch

emilyliublair committed:
add backend folders

emilyliublair committed:
Merge pull request #1 from BoG-Dev-Bootcamp-F23/emily

add backend folders

emilyliublair committed:
add schemas

emilyliublair committed:
Merge pull request #2 from BoG-Dev-Bootcamp-F23/emily

add schemas

Commit:
Merge branch 'main' of https://github.com/BoG-Dev-Bootcamp-F23/project2-f23-cheetah into ankith

emilyliublair committed:
create User schema

emilyliublair committed:
Merge pull request #3 from BoG-Dev-Bootcamp-F23/emily

create User schema

Commit:
Added TrainingLog Schema

Ishaan-Ali committed:
Created animal model

Duckierstone42 committed:
Merge pull request #4 from BoG-Dev-Bootcamp-F23/ankith

Created Training Schema

Ishaan-Ali committed:
Merge branch 'main' of https://github.com/BoG-Dev-Bootcamp-F23/project2-f23-cheetah

Commit:
Changed ref in user and animal to have the same case as the other schema names

Commit:
Changed ref in user and animal to have the same case as the other schema names

Duckierstone42 committed:
Merge pull request #5 from BoG-Dev-Bootcamp-F23/ankith

Changed ref in user and animal to have the same case as the other schema  names

Commit:
Merge branch 'ankith' of https://github.com/BoG-Dev-Bootcamp-F23/project2-f23-cheetah into ankith

Commit:
Moved models into mongodb folder, created index.js, created an actions folder, as well as utils folder

Commit:
Moved models into mongodb folder, created index.js, created an actions folder, as well as utils folder

Commit:
Merge branch 'main' of https://github.com/BoG-Dev-Bootcamp-F23/project2-f23-cheetah

Commit:
Added new errors as well as basic createTrainingLog.js implementation

Commit:
Added new errors as well as basic createTrainingLog.js implementation

Commit:
Added checkss in createTrainingLog to ensure the specified user and animal exist

Commit:
Added checkss in createTrainingLog to ensure the specified user and animal exist

Duckierstone42 committed:
Merge pull request #6 from BoG-Dev-Bootcamp-F23/ankith

Added checks in createTrainingLog to ensure the specified user and animal exists.

Commit:
Created some new API endpoints, added some boilerplate code

Commit:
Created some new API endpoints, added some boilerplate code

emilyliublair committed:
finish user endpoints

emilyliublair committed:
finish user endpoints

emilyliublair committed:
Merge pull request #7 from BoG-Dev-Bootcamp-F23/emily

finish user endpoints

Commit:
Merge branch 'main' into ankith

Commit:
Created all TrainingLog CRUD operations as well as API endpoints. In the future, make sure to add a call to another API to alter the number of animal hours.

Commit:
Created all TrainingLog CRUD operations as well as API endpoints. In the future, make sure to add a call to another API to alter the number of animal hours.

Duckierstone42 committed:
Merge pull request #8 from BoG-Dev-Bootcamp-F23/ankith

Created TrainingLog CRUD methods API Endpoints.

Ishaan-Ali committed:
Finished all backend and CRUD operations for animal endpoints

Ishaan-Ali committed:
Finished all backend and CRUD operations for animal endpoints

Ishaan-Ali committed:
Merge branch 'ishaan' of https://github.com/BoG-Dev-Bootcamp-F23/project2-f23-cheetah into ishaan

Ishaan-Ali committed:
Merge pull request #9 from BoG-Dev-Bootcamp-F23/ishaan

Ishaan

nathangong commented:
@Ishaan-Ali this is the main project repo -- make sure you change the base to your team's repository!

emilyliublair committed:
login + signup pages complete

emilyliublair committed:
Merge pull request #10 from BoG-Dev-Bootcamp-F23/emily

login + signup pages complete

Commit:
Fixed a bunch of issue

Commit:
Fixed minor API issues, standardizes API's a bit, enabled it so that deleting an animal deletes its trainingLogs, and deleting a user deletes everything associated with it.

Commit:
Merge branch 'main' of https://github.com/BoG-Dev-Bootcamp-F23/project2-f23-cheetah into ankith

Duckierstone42 committed:
Merge pull request #11 from BoG-Dev-Bootcamp-F23/ankith

Updated backend

emilyliublair committed:
finish sidebar component (TODO: custom hooks)

emilyliublair committed:
Merge pull request #12 from BoG-Dev-Bootcamp-F23/emily

finish sidebar component (TODO: custom hooks)

Commit:
Created Basic Training Log Implementation

Commit:
Created base implementation of Training page

Duckierstone42 committed:
Merge pull request #13 from BoG-Dev-Bootcamp-F23/ankith

Created Basic training log and training page.

Things TODO:
-Create page for adding and editing elements.
-Implement way to get name and animal name from identifier, like potentially new API endpoint?
-Implement a way to delete training logs
-Add search bar whenever that gets created

Commit:
Created new MongooDB action and endpoint to access username and animal name for traininglog display, as well as made that more interactive.

Commit:
Merge branch 'main' of https://github.com/BoG-Dev-Bootcamp-F23/project2-f23-cheetah into ankith

Duckierstone42 committed:
Merge pull request #14 from BoG-Dev-Bootcamp-F23/ankith

Created new MongooDB action and endpoint to access username and anima…

Commit:
Made the page dynamically display all trainingLogs

Commit:
Implemented basic hashing

Commit:
Incorrect hashing attempt

Commit:
Revert "Incorrect hashing attempt"

This reverts commit c0f87c31c295c9fcdf2b20fd9aebef64e1544cae.

Commit:
Implemented hashing

Duckierstone42 committed:
Merge pull request #15 from BoG-Dev-Bootcamp-F23/ankith

Implemented hashing

Ishaan-Ali committed:
Fixed issue with getAnimals

Ishaan-Ali committed:
Merge pull request #16 from BoG-Dev-Bootcamp-F23/ishaan

Fixed issue with getAnimals

Commit:
Created first part of create page for the Training Log

Commit:
Created working Animal Selection options

Commit:
Made functional Create button

Commit:
Create mechanism now fully functional

Ishaan-Ali committed:
Animal page additions

Commit:
Created working implementation of Training Log edit

Commit:
Fully finished all functional stuff for create and edit training log, simply have one error with rendering when you switch the animal

Commit:
Merge branch 'main' of https://github.com/BoG-Dev-Bootcamp-F23/project2-f23-cheetah into ankith

Duckierstone42 committed:
Merge pull request #17 from BoG-Dev-Bootcamp-F23/ankith

Implemented Functional behavior for creating and editing training logs

Ishaan-Ali committed:
Merge branch 'main' of https://github.com/BoG-Dev-Bootcamp-F23/project2-f23-cheetah into ishaan

Ishaan-Ali committed:
current progress on Animal page, need to fix styling and stuff, implement create/edit/delete operations

Ishaan-Ali committed:
Merge pull request #18 from BoG-Dev-Bootcamp-F23/ishaan

Ishaan

Commit:
Merge branch 'main' of https://github.com/BoG-Dev-Bootcamp-F23/project2-f23-cheetah into ankith

Duckierstone42 committed:
Delete .env

There shouldn't be an .env file in the repo, make sure to keep it in your .gitignore.

Commit:
Created basic CSS styling for create page

emilyliublair committed:
implement custom hooks, loading screen, & multiple email check

emilyliublair committed:
Merge pull request #19 from BoG-Dev-Bootcamp-F23/emily

implement custom hooks, loading screen, & multiple email check

Commit:
Merge branch 'main' of https://github.com/BoG-Dev-Bootcamp-F23/project2-f23-cheetah into ankith

Commit:
Merge branch 'main' of https://github.com/BoG-Dev-Bootcamp-F23/project2-f23-cheetah into ankith

Commit:
Added basic create page css and enabled traininglog page to use userId.

Commit:
Implemented functional search bar for Training Log (Still need to add styling)

Duckierstone42 committed:
Merge pull request #20 from BoG-Dev-Bootcamp-F23/ankith

Ankith

Duckierstone42 committed:
Delete .env

Commit:
Created Titlebar and added it to all current pages

Duckierstone42 committed:
Merge pull request #21 from BoG-Dev-Bootcamp-F23/ankith

Created Titlebar

Commit:
Created admin  trainingpage at /traininglogsadmin

Commit:
Made sidebar interactive

Duckierstone42 committed:
Merge pull request #22 from BoG-Dev-Bootcamp-F23/ankith

Made sidebar interactive

Commit:
Added hover effects for sidebar and trainingpage

Commit:
Updated grid layout for animals

Duckierstone42 committed:
Merge pull request #23 from BoG-Dev-Bootcamp-F23/ankith

Updated Animals page layout

Commit:
Added scrolling for traininglogs and animals

emilyliublair committed:
create cookie on login w jwt

emilyliublair committed:
Merge branch 'main' into emily

emilyliublair committed:
Merge pull request #24 from BoG-Dev-Bootcamp-F23/emily

create cookie on login w jwt

emilyliublair committed:
comment out cookie stuff

Duckierstone42 committed:
Merge pull request #25 from BoG-Dev-Bootcamp-F23/ankith

Scrolling bar

emilyliublair committed:
refactor code + allusers component

emilyliublair committed:
Merge pull request #26 from BoG-Dev-Bootcamp-F23/emily

refactor code + allusers component

emilyliublair committed:
add alltraininglogs functionality

emilyliublair committed:
Merge pull request #27 from BoG-Dev-Bootcamp-F23/emily

add alltraininglogs functionality

Commit:
Made search bar work again

Commit:
Made create button work again

Duckierstone42 committed:
Merge pull request #28 from BoG-Dev-Bootcamp-F23/ankith

Fixed search bar and create button

emilyliublair committed:
add animal display pgs

emilyliublair committed:
Merge branch 'main' into emily

emilyliublair committed:
Merge pull request #29 from BoG-Dev-Bootcamp-F23/emily

add animal display pgs

emilyliublair committed:
fix animal display

emilyliublair committed:
Merge pull request #30 from BoG-Dev-Bootcamp-F23/emily

fix animal display

Commit:
Enabled searching for all pages

Commit:
Fixed seaching

Duckierstone42 committed:
Merge pull request #31 from BoG-Dev-Bootcamp-F23/ankith

Made searching working for everything

Commit:
Fixed merge conflicts

Commit:
Merge branch 'main' of https://github.com/BoG-Dev-Bootcamp-F23/project2-f23-cheetah into ankith

Duckierstone42 committed:
Merge pull request #32 from BoG-Dev-Bootcamp-F23/ankith

Fixed merge conflicts

emilyliublair committed:
create cookie

emilyliublair committed:
Merge pull request #33 from BoG-Dev-Bootcamp-F23/emily

create cookie

Ishaan-Ali committed:
Added animal changes

Ishaan-Ali committed:
Added animal changes

Commit:
Removed verify changes that broke the code

Commit:
Merge branch 'main' of https://github.com/BoG-Dev-Bootcamp-F23/project2-f23-cheetah into ankith

Ishaan-Ali committed:
Added animal changes, create animal not working

Ishaan-Ali committed:
Merge branch 'ankith' of https://github.com/BoG-Dev-Bootcamp-F23/project2-f23-cheetah into ankith

Ishaan-Ali committed:
More animal changes

Ishaan-Ali committed:
Fixed issue with animal search

Commit:
Readded verify.js page that works

Commit:
Merge branch 'ankith' of https://github.com/BoG-Dev-Bootcamp-F23/project2-f23-cheetah into ankith

Commit:
Added scrolling and all users to userpage

Duckierstone42 committed:
Merge pull request #34 from BoG-Dev-Bootcamp-F23/ankith

Added scrolling on all uses page, also made it display all users

Commit:
Fixed some small changes to animal page

Duckierstone42 committed:
Merge pull request #35 from BoG-Dev-Bootcamp-F23/ankith

Fixed some small changes to animal page

Commit:
Fixed some more issues with animal list

Duckierstone42 committed:
Merge pull request #36 from BoG-Dev-Bootcamp-F23/ankith

Fixed some more issues with animal list

Commit:
Fixed animal styling further

Commit:
Removed some unecessary code

Duckierstone42 committed:
Merge pull request #37 from BoG-Dev-Bootcamp-F23/ankith

Ankith

Commit:
Fixed create button being off screen and weird animal card size

Duckierstone42 committed:
Merge pull request #38 from BoG-Dev-Bootcamp-F23/ankith

Fixed create button being off screen and weird animal card size

Ishaan-Ali committed:
Merge branch 'main' of https://github.com/BoG-Dev-Bootcamp-F23/project2-f23-cheetah into ishaan

Commit:
Fixed issue where not all animals were getting displayed

Duckierstone42 committed:
Merge pull request #39 from BoG-Dev-Bootcamp-F23/ankith

Fixed issue where not all animals were getting displayed

Ishaan-Ali committed:
Merge branch 'main' of https://github.com/BoG-Dev-Bootcamp-F23/project2-f23-cheetah into ishaan

Commit:
Added red circle to the bottom left

Commit:
Made searchbar clear when switching options

Commit:
Added a default dog image

Commit:
Fixed search bar problem

Duckierstone42 committed:
Merge pull request #40 from BoG-Dev-Bootcamp-F23/ankith

Ankith

Commit:
Fixed issue with login page redirecting to non-existing page

Ishaan-Ali committed:
Fixed styling issue with edit

Commit:
Added delete to user, when you press the user profile pic. I will ad a seperate delete button later

Duckierstone42 committed:
Merge pull request #41 from BoG-Dev-Bootcamp-F23/ankith

Added user delete option

emilyliublair committed:
jwt + cookie functionality complete

emilyliublair committed:
Merge branch 'main' into emily

emilyliublair committed:
Merge pull request #42 from BoG-Dev-Bootcamp-F23/emily

jwt + cookie functionality complete

Commit:
Added delete option for users

Duckierstone42 committed:
Merge pull request #43 from BoG-Dev-Bootcamp-F23/ankith

Added front-end delete option for users

Ishaan-Ali committed:
Added styling changes to edit

Ishaan-Ali committed:
Merge branch 'main' of https://github.com/BoG-Dev-Bootcamp-F23/project2-f23-cheetah into ishaan

Ishaan-Ali committed:
Merge pull request #44 from BoG-Dev-Bootcamp-F23/ishaan

Added styling changes

Ishaan-Ali committed:
Merge branch 'main' of https://github.com/BoG-Dev-Bootcamp-F23/project2-f23-cheetah into ishaan

Ishaan-Ali committed:
Added styling to save cancel and delete buttons
